kms/snippets_test.go

```diff
@@ -336,7 +337,7 @@ func TestSymmetricEncryptDecrypt(t *testing.T) {
 		t.Fatalf("decrypted plaintext does not match input message: want %s, got %s", []byte(v.message), plainBytes)
 	}
 	if bytes.Equal(cipherBytes, []byte(v.message)) {
-		t.Fatalf("ciphertext and plaintext bytes are identical: %s", cipherBytes)
+		t.Fatalf("ciphertext and plaintext bytes are IDentical: %s", cipherBytes)
```

Comment:
looks like this was mistakenly caught in find/replace

Reply:
Whoops! Done. Thanks!